vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ApplicationSteps.java

```diff
@@ -90,14 +90,36 @@ public void startMobileApplication()
     }
 
     /**
-     * Closes mobile application
+     * Closes mobile application and quits the session
      */
     @When("I close mobile application")
     public void closeMobileApplication()
     {
         webDriverProvider.end();
     }
 
+    /**
+     * Restart mobile application
```

Comment:
Steps wording mentions reinstall, javadoc says about restart, looks confusing for the end user